db/seed_csvs/markets.csv

```diff
@@ -1,3 +1,4 @@
+fake_id,name,address,city,county,state,zip
```

Comment:
how did adding the first line`fake_id,name,address,city,county,state,zip` to this file affect your data? how did this affect the relationships between markets & vendors?

take a look at how active record sets up the relationships and uses of `id` for you when you're doing `has_many` and `belongs_to` relationships. [http://guides.rubyonrails.org/association_basics.html#the-has-many-association](Ruby on Rails Guide)